### ISSUE TITLE
'CAG' is not exported by node_modules/@jscad/csg/csg.js 

### ISSUE THREAD

**Issue by gfwilliams** [jscad/scad-api#8]
As referenced on https://github.com/jscad/scad-api/pull/7

This was while trying to build scad-api using `npm run build`

My guess is I actually need to pull in the CSG library from this project as well. Just pushing a newer version to NPM might fix it?

```
gw@edgey ~/workspace $ git clone https://github.com/jscad/scad-api
Cloning into 'scad-api'...
remote: Counting objects: 237, done.
remote: Total 237 (delta 0), reused 0 (delta 0), pack-reused 237
Receiving objects: 100% (237/237), 110.16 KiB | 0 bytes/s, done.
Resolving deltas: 100% (130/130), done.
Checking connectivity... done.
gw@edgey ~/workspace $ cd scad-api/
gw@edgey ~/workspace/scad-api $ npm install
@jscad/scad-api@0.1.0 /home/gw/workspace/scad-api
├── @jscad/csg@0.1.4 
├─┬ ava@0.15.2 
│ ├── arr-diff@2.0.0 
│ ├── arr-flatten@1.0.1 
│ ├── array-union@1.0.2 
│ ├── array-uniq@1.0.3 
│ ├── arrify@1.0.1 
│ ├─┬ ava-init@0.1.6 
│ │ ├── arr-exclude@1.0.0 
│ │ ├─┬ cross-spawn@4.0.2 
│ │ │ ├─┬ lru-cache@4.0.2 
│ │ │ │ ├── pseudomap@1.0.2 
│ │ │ │ └── yallist@2.1.2 
│ │ │ └─┬ which@1.2.12 
│ │ │   └── isexe@1.1.2 
│ │ ├─┬ pinkie-promise@2.0.1 
│ │ │ └── pinkie@2.0.4 
│ │ ├─┬ read-pkg-up@1.0.1 
│ │ │ └─┬ read-pkg@1.1.0 
│ │ │   └── path-type@1.1.0 
│ │ ├── the-argv@1.0.0 
│ │ └─┬ write-pkg@1.0.0 
│ │   └─┬ write-json-file@1.2.0 
│ │     └─┬ sort-keys@1.1.2 
│ │       └── is-plain-obj@1.1.0 
│ ├─┬ babel-code-frame@6.22.0 
│ │ ├── esutils@2.0.2 
│ │ └── js-tokens@3.0.1 
│ ├─┬ babel-core@6.24.0 
│ │ ├─┬ babel-generator@6.24.0 
│ │ │ ├── detect-indent@4.0.0 
│ │ │ ├── jsesc@1.3.0 
│ │ │ └── trim-right@1.0.1 
│ │ ├── babel-helpers@6.23.0 
│ │ ├── babel-messages@6.23.0 
│ │ ├── babel-template@6.23.0 
│ │ ├─┬ babel-traverse@6.23.1 
│ │ │ ├── globals@9.16.0 
│ │ │ └─┬ invariant@2.2.2 
│ │ │   └── loose-envify@1.3.1 
│ │ ├─┬ babel-types@6.23.0 
│ │ │ └── to-fast-properties@1.0.2 
│ │ ├── babylon@6.16.1 
│ │ ├── json5@0.5.1 
│ │ ├─┬ minimatch@3.0.3 
│ │ │ └─┬ brace-expansion@1.1.6 
│ │ │   ├── balanced-match@0.4.2 
│ │ │   └── concat-map@0.0.1 
│ │ ├── path-is-absolute@1.0.1 
│ │ ├── private@0.1.7 
│ │ └── source-map@0.5.6 
│ ├── babel-plugin-ava-throws-helper@0.0.4 
│ ├── babel-plugin-detective@1.0.2 
│ ├─┬ babel-plugin-espower@2.3.2 
│ │ ├─┬ call-matcher@1.0.1 
│ │ │ └── deep-equal@1.0.1 
│ │ ├─┬ espower-location-detector@1.0.0 
│ │ │ └── is-url@1.2.2 
│ │ ├── espurify@1.7.0 
│ │ └── estraverse@4.2.0 
│ ├── babel-plugin-transform-runtime@6.23.0 
│ ├─┬ babel-preset-stage-2@6.22.0 
│ │ ├── babel-plugin-syntax-dynamic-import@6.18.0 
│ │ ├─┬ babel-plugin-transform-class-properties@6.23.0 
│ │ │ └── babel-plugin-syntax-class-properties@6.13.0 
│ │ ├─┬ babel-plugin-transform-decorators@6.22.0 
│ │ │ ├─┬ babel-helper-explode-class@6.22.0 
│ │ │ │ └── babel-helper-bindify-decorators@6.22.0 
│ │ │ └── babel-plugin-syntax-decorators@6.13.0 
│ │ └─┬ babel-preset-stage-3@6.22.0 
│ │   ├── babel-plugin-syntax-trailing-function-commas@6.22.0 
│ │   ├─┬ babel-plugin-transform-async-generator-functions@6.22.0 
│ │   │ ├── babel-helper-remap-async-to-generator@6.22.0 
│ │   │ └── babel-plugin-syntax-async-generators@6.13.0 
│ │   ├─┬ babel-plugin-transform-async-to-generator@6.22.0 
│ │   │ └── babel-plugin-syntax-async-functions@6.13.0 
│ │   ├─┬ babel-plugin-transform-exponentiation-operator@6.22.0 
│ │   │ ├─┬ babel-helper-builder-binary-assignment-operator-visitor@6.22.0 
│ │   │ │ └── babel-helper-explode-assignable-expression@6.22.0 
│ │   │ └── babel-plugin-syntax-exponentiation-operator@6.13.0 
│ │   └─┬ babel-plugin-transform-object-rest-spread@6.23.0 
│ │     └── babel-plugin-syntax-object-rest-spread@6.13.0 
│ ├─┬ babel-runtime@6.23.0 
│ │ └── regenerator-runtime@0.10.3 
│ ├── bluebird@3.5.0 
│ ├─┬ caching-transform@1.0.1 
│ │ └─┬ write-file-atomic@1.3.1 
│ │   ├── graceful-fs@4.1.11 
│ │   ├── imurmurhash@0.1.4 
│ │   └── slide@1.1.6 
│ ├─┬ chalk@1.1.3 
│ │ ├── ansi-styles@2.2.1 
│ │ ├── escape-string-regexp@1.0.5 
│ │ ├── has-ansi@2.0.0 
│ │ └── supports-color@2.0.0 
│ ├─┬ chokidar@1.6.1 
│ │ ├─┬ anymatch@1.3.0 
│ │ │ └─┬ micromatch@2.3.11 
│ │ │   ├── array-unique@0.2.1 
│ │ │   ├─┬ braces@1.8.5 
│ │ │   │ ├─┬ expand-range@1.8.2 
│ │ │   │ │ └─┬ fill-range@2.2.3 
│ │ │   │ │   ├── is-number@2.1.0 
│ │ │   │ │   ├── isobject@2.1.0 
│ │ │   │ │   ├── randomatic@1.1.6 
│ │ │   │ │   └── repeat-string@1.6.1 
│ │ │   │ ├── preserve@0.2.0 
│ │ │   │ └── repeat-element@1.1.2 
│ │ │   ├─┬ expand-brackets@0.1.5 
│ │ │   │ └── is-posix-bracket@0.1.1 
│ │ │   ├── extglob@0.3.2 
│ │ │   ├── filename-regex@2.0.0 
│ │ │   ├─┬ kind-of@3.1.0 
│ │ │   │ └── is-buffer@1.1.5 
│ │ │   ├── normalize-path@2.0.1 
│ │ │   ├─┬ object.omit@2.0.1 
│ │ │   │ ├─┬ for-own@0.1.5 
│ │ │   │ │ └── for-in@1.0.2 
│ │ │   │ └── is-extendable@0.1.1 
│ │ │   ├─┬ parse-glob@3.0.4 
│ │ │   │ ├── glob-base@0.3.0 
│ │ │   │ └── is-dotfile@1.0.2 
│ │ │   └─┬ regex-cache@0.4.3 
│ │ │     ├── is-equal-shallow@0.1.3 
│ │ │     └── is-primitive@2.0.0 
│ │ ├── async-each@1.0.1 
│ │ ├── glob-parent@2.0.0 
│ │ ├── inherits@2.0.3 
│ │ ├─┬ is-binary-path@1.0.1 
│ │ │ └── binary-extensions@1.8.0 
│ │ ├─┬ is-glob@2.0.1 
│ │ │ └── is-extglob@1.0.0 
│ │ └─┬ readdirp@2.1.0 
│ │   └─┬ readable-stream@2.2.3 
│ │     ├── buffer-shims@1.0.0 
│ │     ├── core-util-is@1.0.2 
│ │     ├── isarray@1.0.0 
│ │     ├── process-nextick-args@1.0.7 
│ │     ├── string_decoder@0.10.31 
│ │     └── util-deprecate@1.0.2 
│ ├── clean-yaml-object@0.1.0 
│ ├─┬ cli-cursor@1.0.2 
│ │ └─┬ restore-cursor@1.0.1 
│ │   ├── exit-hook@1.1.1 
│ │   └── onetime@1.1.0 
│ ├── cli-spinners@0.1.2 
│ ├─┬ cli-truncate@0.2.1 
│ │ ├── slice-ansi@0.0.4 
│ │ └─┬ string-width@1.0.2 
│ │   ├── code-point-at@1.1.0 
│ │   └── is-fullwidth-code-point@1.0.0 
│ ├─┬ co-with-promise@4.6.0 
│ │ └─┬ pinkie-promise@1.0.0 
│ │   └── pinkie@1.0.0 
│ ├── common-path-prefix@1.0.0 
│ ├── convert-source-map@1.4.0 
│ ├─┬ core-assert@0.2.1 
│ │ ├── buf-compare@1.0.1 
│ │ └── is-error@2.2.1 
│ ├─┬ currently-unhandled@0.4.1 
│ │ └── array-find-index@1.0.2 
│ ├─┬ debug@2.6.3 
│ │ └── ms@0.7.2 
│ ├─┬ empower-core@0.5.0 
│ │ ├── call-signature@0.0.2 
│ │ ├── core-js@1.2.7 
│ │ └── xtend@4.0.1 
│ ├── figures@1.7.0 
│ ├─┬ find-cache-dir@0.1.1 
│ │ ├── commondir@1.0.1 
│ │ └── pkg-dir@1.0.0 
│ ├── fn-name@2.0.1 
│ ├─┬ globby@4.1.0 
│ │ ├─┬ glob@6.0.4 
│ │ │ ├─┬ inflight@1.0.6 
│ │ │ │ └── wrappy@1.0.2 
│ │ │ └── once@1.4.0 
│ │ └── pify@2.3.0 
│ ├── has-flag@2.0.0 
│ ├── ignore-by-default@1.0.1 
│ ├─┬ is-ci@1.0.10 
│ │ └── ci-info@1.0.0 
│ ├── is-generator-fn@1.0.0 
│ ├── is-obj@1.0.1 
│ ├─┬ is-observable@0.2.0 
│ │ └── symbol-observable@0.2.4 
│ ├── is-promise@2.1.0 
│ ├─┬ last-line-stream@1.0.0 
│ │ └── through2@2.0.3 
│ ├── lodash.debounce@4.0.8 
│ ├─┬ loud-rejection@1.6.0 
│ │ └── signal-exit@3.0.2 
│ ├── matcher@0.1.2 
│ ├── max-timeout@1.0.0 
│ ├─┬ md5-hex@1.3.0 
│ │ └── md5-o-matic@0.1.1 
│ ├─┬ meow@3.7.0 
│ │ ├─┬ camelcase-keys@2.1.0 
│ │ │ └── camelcase@2.1.1 
│ │ ├── decamelize@1.2.0 
│ │ ├── map-obj@1.0.1 
│ │ ├── minimist@1.2.0 
│ │ ├─┬ normalize-package-data@2.3.6 
│ │ │ ├── hosted-git-info@2.2.0 
│ │ │ ├── is-builtin-module@1.0.0 
│ │ │ ├── semver@5.3.0 
│ │ │ └─┬ validate-npm-package-license@3.0.1 
│ │ │   ├─┬ spdx-correct@1.0.2 
│ │ │   │ └── spdx-license-ids@1.2.2 
│ │ │   └── spdx-expression-parse@1.0.4 
│ │ ├─┬ redent@1.0.0 
│ │ │ ├── indent-string@2.1.0 
│ │ │ └─┬ strip-indent@1.0.1 
│ │ │   └── get-stdin@4.0.1 
│ │ └── trim-newlines@1.0.0 
│ ├── ms@0.7.3 
│ ├─┬ multimatch@2.1.0 
│ │ └── array-differ@1.0.0 
│ ├─┬ not-so-shallow@0.1.4 
│ │ └── buffer-equals@1.0.4 
│ ├── object-assign@4.1.1 
│ ├── observable-to-promise@0.4.0 
│ ├── option-chain@0.1.1 
│ ├── package-hash@1.2.0 
│ ├─┬ pkg-conf@1.1.3 
│ │ ├─┬ find-up@1.1.2 
│ │ │ └── path-exists@2.1.0 
│ │ ├─┬ load-json-file@1.1.0 
│ │ │ └─┬ parse-json@2.2.0 
│ │ │   └─┬ error-ex@1.3.1 
│ │ │     └── is-arrayish@0.2.1 
│ │ └── symbol@0.2.3 
│ ├─┬ plur@2.1.2 
│ │ └── irregular-plurals@1.2.0 
│ ├─┬ power-assert-formatter@1.4.1 
│ │ ├─┬ power-assert-context-formatter@1.1.1 
│ │ │ └── power-assert-context-traversal@1.1.1 
│ │ ├─┬ power-assert-context-reducer-ast@1.1.1 
│ │ │ └── acorn-es7-plugin@1.1.5 
│ │ ├─┬ power-assert-renderer-assertion@1.1.1 
│ │ │ ├── power-assert-renderer-base@1.1.1 
│ │ │ └─┬ power-assert-util-string-width@1.1.1 
│ │ │   └── eastasianwidth@0.1.1 
│ │ ├─┬ power-assert-renderer-comparison@1.1.1 
│ │ │ ├── diff-match-patch@1.0.0 
│ │ │ ├─┬ stringifier@1.3.0 
│ │ │ │ └── traverse@0.6.6 
│ │ │ └── type-name@2.0.2 
│ │ ├── power-assert-renderer-diagram@1.1.1 
│ │ └── power-assert-renderer-file@1.1.1 
│ ├─┬ power-assert-renderers@0.1.1 
│ │ └── power-assert-renderer-succinct@1.1.1 
│ ├─┬ pretty-ms@2.1.0 
│ │ ├─┬ is-finite@1.0.2 
│ │ │ └── number-is-nan@1.0.1 
│ │ ├── parse-ms@1.0.1 
│ │ └── plur@1.0.0 
│ ├── repeating@2.0.1 
│ ├── require-precompiled@0.1.0 
│ ├─┬ resolve-cwd@1.0.0 
│ │ └── resolve-from@2.0.0 
│ ├── set-immediate-shim@1.0.1 
│ ├── slash@1.0.0 
│ ├── source-map-support@0.4.12 
│ ├── stack-utils@0.4.0 
│ ├─┬ strip-ansi@3.0.1 
│ │ └── ansi-regex@2.1.1 
│ ├─┬ strip-bom@2.0.0 
│ │ └── is-utf8@0.2.1 
│ ├─┬ time-require@0.1.2 
│ │ ├─┬ chalk@0.4.0 
│ │ │ ├── ansi-styles@1.0.0 
│ │ │ ├── has-color@0.1.7 
│ │ │ └── strip-ansi@0.1.1 
│ │ ├── date-time@0.1.1 
│ │ ├─┬ pretty-ms@0.2.2 
│ │ │ └── parse-ms@0.1.2 
│ │ └── text-table@0.2.0 
│ ├─┬ unique-temp-dir@1.0.0 
│ │ ├── os-tmpdir@1.0.2 
│ │ └── uid2@0.0.3 
│ └─┬ update-notifier@0.7.0 
│   ├── ansi-align@1.1.0 
│   ├─┬ boxen@0.5.1 
│   │ ├── cli-boxes@1.0.0 
│   │ ├── filled-array@1.1.0 
│   │ └── widest-line@1.0.0 
│   ├─┬ configstore@2.1.0 
│   │ ├── dot-prop@3.0.0 
│   │ ├── osenv@0.1.4 
│   │ └── uuid@2.0.3 
│   ├── is-npm@1.0.0 
│   ├─┬ latest-version@2.0.0 
│   │ └─┬ package-json@2.4.0 
│   │   ├─┬ got@5.7.1 
│   │   │ ├─┬ create-error-class@3.0.2 
│   │   │ │ └── capture-stack-trace@1.0.0 
│   │   │ ├── duplexer2@0.1.4 
│   │   │ ├── is-redirect@1.0.0 
│   │   │ ├── is-retry-allowed@1.1.0 
│   │   │ ├── is-stream@1.1.0 
│   │   │ ├── lowercase-keys@1.0.0 
│   │   │ ├── node-status-codes@1.0.0 
│   │   │ ├── read-all-stream@3.1.0 
│   │   │ ├── timed-out@3.1.3 
│   │   │ ├── unzip-response@1.0.2 
│   │   │ └─┬ url-parse-lax@1.0.0 
│   │   │   └── prepend-http@1.0.4 
│   │   ├─┬ registry-auth-token@3.1.0 
│   │   │ └─┬ rc@1.1.7 
│   │   │   ├── deep-extend@0.4.1 
│   │   │   ├── ini@1.3.4 
│   │   │   ├── minimist@1.2.0 
│   │   │   └── strip-json-comments@2.0.1 
│   │   └── registry-url@3.1.0 
│   ├── semver-diff@2.1.0 
│   └── xdg-basedir@2.0.0 
├─┬ babel-preset-es2015@6.24.0 
│ ├── babel-plugin-check-es2015-constants@6.22.0 
│ ├── babel-plugin-transform-es2015-arrow-functions@6.22.0 
│ ├── babel-plugin-transform-es2015-block-scoped-functions@6.22.0 
│ ├── babel-plugin-transform-es2015-block-scoping@6.23.0 
│ ├─┬ babel-plugin-transform-es2015-classes@6.23.0 
│ │ ├── babel-helper-define-map@6.23.0 
│ │ ├── babel-helper-function-name@6.23.0 
│ │ ├── babel-helper-optimise-call-expression@6.23.0 
│ │ └── babel-helper-replace-supers@6.23.0 
│ ├── babel-plugin-transform-es2015-computed-properties@6.22.0 
│ ├── babel-plugin-transform-es2015-destructuring@6.23.0 
│ ├── babel-plugin-transform-es2015-duplicate-keys@6.22.0 
│ ├── babel-plugin-transform-es2015-for-of@6.23.0 
│ ├── babel-plugin-transform-es2015-function-name@6.22.0 
│ ├── babel-plugin-transform-es2015-literals@6.22.0 
│ ├── babel-plugin-transform-es2015-modules-amd@6.24.0 
│ ├─┬ babel-plugin-transform-es2015-modules-commonjs@6.24.0 
│ │ └── babel-plugin-transform-strict-mode@6.22.0 
│ ├─┬ babel-plugin-transform-es2015-modules-systemjs@6.23.0 
│ │ └── babel-helper-hoist-variables@6.22.0 
│ ├── babel-plugin-transform-es2015-modules-umd@6.24.0 
│ ├── babel-plugin-transform-es2015-object-super@6.22.0 
│ ├─┬ babel-plugin-transform-es2015-parameters@6.23.0 
│ │ ├── babel-helper-call-delegate@6.22.0 
│ │ └── babel-helper-get-function-arity@6.22.0 
│ ├── babel-plugin-transform-es2015-shorthand-properties@6.22.0 
│ ├── babel-plugin-transform-es2015-spread@6.22.0 
│ ├─┬ babel-plugin-transform-es2015-sticky-regex@6.22.0 
│ │ └── babel-helper-regex@6.22.0 
│ ├── babel-plugin-transform-es2015-template-literals@6.22.0 
│ ├── babel-plugin-transform-es2015-typeof-symbol@6.23.0 
│ ├─┬ babel-plugin-transform-es2015-unicode-regex@6.22.0 
│ │ └─┬ regexpu-core@2.0.0 
│ │   ├── regenerate@1.3.2 
│ │   ├── regjsgen@0.2.0 
│ │   └─┬ regjsparser@0.1.5 
│ │     └── jsesc@0.5.0 
│ └─┬ babel-plugin-transform-regenerator@6.22.0 
│   └── regenerator-transform@0.9.8 
├─┬ babel-register@6.24.0 
│ ├── core-js@2.4.1 
│ ├─┬ home-or-tmp@2.0.0 
│ │ └── os-homedir@1.0.2 
│ ├── lodash@4.17.4 
│ └─┬ mkdirp@0.5.1 
│   └── minimist@0.0.8 
├── rollup@0.38.3 
├─┬ rollup-plugin-buble@0.15.0 
│ ├─┬ buble@0.15.2 
│ │ ├── acorn@3.3.0 
│ │ ├─┬ acorn-jsx@3.0.1 
│ │ │ └── acorn@3.3.0 
│ │ ├─┬ acorn-object-spread@1.0.0 
│ │ │ └── acorn@3.3.0 
│ │ ├── magic-string@0.14.0 
│ │ └── minimist@1.2.0 
│ └─┬ rollup-pluginutils@1.5.2 
│   └── estree-walker@0.2.1 
├─┬ rollup-plugin-commonjs@6.0.1 
│ ├── acorn@4.0.11 
│ ├── estree-walker@0.3.1 
│ ├─┬ magic-string@0.19.0 
│ │ └── vlq@0.2.1 
│ └─┬ resolve@1.3.2 
│   └── path-parse@1.0.5 
└─┬ rollup-plugin-node-resolve@2.0.0 
  ├─┬ browser-resolve@1.11.2 
  │ └── resolve@1.1.7 
  └── builtin-modules@1.1.1 

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
gw@edgey ~/workspace/scad-api $ npm run build

> @jscad/scad-api@0.1.0 build /home/gw/workspace/scad-api
> rollup -c rollup.config.js

'CAG' is not exported by node_modules/@jscad/csg/csg.js (imported by src/index.js). For help fixing this error see https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module
Error: 'CAG' is not exported by node_modules/@jscad/csg/csg.js (imported by src/index.js). For help fixing this error see https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module
    at error (/home/gw/workspace/scad-api/node_modules/rollup/src/utils/error.js:2:14)
    at Module.trace (/home/gw/workspace/scad-api/node_modules/rollup/src/Module.js:361:4)
    at ModuleScope.findDeclaration (/home/gw/workspace/scad-api/node_modules/rollup/src/ast/scopes/ModuleScope.js:48:22)
    at Node.bind (/home/gw/workspace/scad-api/node_modules/rollup/src/ast/nodes/Identifier.js:22:29)
    at /home/gw/workspace/scad-api/node_modules/rollup/src/ast/Node.js:6:34
    at Node.eachChild (/home/gw/workspace/scad-api/node_modules/rollup/src/ast/Node.js:21:5)
    at Node.bind (/home/gw/workspace/scad-api/node_modules/rollup/src/ast/Node.js:6:8)
    at /home/gw/workspace/scad-api/node_modules/rollup/src/ast/Node.js:6:34
    at Node.eachChild (/home/gw/workspace/scad-api/node_modules/rollup/src/ast/Node.js:18:20)
    at Node.bind (/home/gw/workspace/scad-api/node_modules/rollup/src/ast/Node.js:6:8)
Type rollup --help for help, or visit https://github.com/rollup/rollup/wiki

npm ERR! Linux 4.4.0-62-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "build"
npm ERR! node v7.7.1
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! @jscad/scad-api@0.1.0 build: `rollup -c rollup.config.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @jscad/scad-api@0.1.0 build script 'rollup -c rollup.config.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the @jscad/scad-api package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rollup -c rollup.config.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs @jscad/scad-api
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls @jscad/scad-api
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/gw/workspace/scad-api/npm-debug.log

```

**Comment by z3dev** [comment]
I hit this to when trying to 'npm run build'
Any hints as to why this is happening?

**Comment by kaosat-dev** [comment]
this is now fixed in (960f6867d8e78149b4c3daf28bb01a1e2d8d8e8a), it was due to the rollup file still referencing the github version of CSG.js & not the one from NPM  , sorry about the delay folks :)
